guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -4,43 +4,71 @@
 {ProjectName} can import content from the Red{nbsp}Hat Content Delivery Network (CDN).
 {Project} requires a Red{nbsp}Hat subscription manifest to find, access, and download content from the corresponding repositories.
 You must have a Red{nbsp}Hat subscription manifest containing a subscription allocation for each organization on {ProjectServer}.
-All subscription information is available in your Red Hat Customer Portal account.
+All subscription information is available on the {RHCloud}.
 
 Use this chapter to import a Red{nbsp}Hat subscription manifest and manage the manifest within the {ProjectWebUI}.
 
 .Subscription allocations and organizations
-
 You can manage more than one organization if you have more than one subscription allocation.
 {Project} requires a single allocation for each organization configured in {ProjectServer}.
-The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red Hat accounts.
+The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red{nbsp}Hat accounts.
 
 .Future-dated subscriptions
-
 You can use future-dated subscriptions in a subscription manifest.
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
+.Subscriptions service capabilities
+The Subscriptions service is a service on the {RHCloud} that helps organizations track and manage their Red{nbsp}Hat subscription usage across connected and disconnected environments. 
+
+* Allows customers to view subscription usage.
+* When configured, it displays usage data from both connected and disconnected environments.
+* Tracks Red{nbsp}Hat product usage and capacity at the organization or account level.
+* Generates a daily snapshot of usage data across all tracked systems.
+* Displays data in a filterable dashboard at the {RHCloud}.
+* Collects historical data for analytics, purchasing decisions, capacity planning, and risk mitigation.
+* Supports the transition to Simple Content Access (SCA), which enforces subscriptions across the organization rather than on individual systems.
```

Comment:
I would either

* Move the capabilities into the procedure module, or
* Link to the procedure module from here

Please, consider which option is more suitable. I'm not sure. Perhaps we should say something about the Subscriptions service in this concept module, but it feels like the details should stick together with the procedure.

Reply:
I'm rewriting the capabilities as an intro without a list in the concept and then linking to the procedure.